packages/modules/src/modules/banners/choiceCardsButtonsBanner/components/ChoiceCards.tsx

```diff
@@ -80,7 +79,7 @@ const styles = {
             border-radius: 10px;
         }
         legend {
-            ${visuallyHidden};
+            ${'\n\topacity: 0;\n\theight: 0;\n\twidth: 0;\n\ttop: 0;\n\tleft: 0;\n;'};
```

Comment:
Hi 👋 , can this line be replaced with some standard css lines? eg:

```
opacity: 0;
height: 0;
width: 0;
left: 0;
```

Reply:
Yep, good call Richard, much neater, thanks!